test/common/http/codes_test.cc

```diff
@@ -154,8 +155,7 @@ TEST_F(CodeUtilityTest, All) {
   const std::vector<std::pair<Code, std::string>> test_set = {
       std::make_pair(Code::Continue, "Continue"),
       std::make_pair(Code::SwitchingProtocols, "Switching Protocols"),
-      std::make_pair(Code::OK, "OK"),
-      std::make_pair(Code::Created, "Created"),
+      std::make_pair(Code::OK, "OK"), std::make_pair(Code::Created, "Created"),
```

Comment:
Maybe there's a comment that can tell clang-format to fuck off and not rearrange this whole block, because this change sucks?